src/apps/reservation-list/modal/pause-reservation/date-input.tsx

```diff
@@ -0,0 +1,34 @@
+import React, { FC } from "react";
+
+interface DateInputProps {
+  label: string;
+  id: string;
+  value: string;
+  onChange: (date: string) => void;
+}
+
+const DateInput: FC<DateInputProps> = ({ label, id, value, onChange }) => {
```

Comment:
This is very generic, and that is cool. But I think we should move it to a global place then.

Reply:
moved it